.gitignore

```diff
@@ -14,3 +14,5 @@ website/node_modules
 website/package-lock.json
 website/translated_docs
 website/yarn.lock
+
+.idea/
```

Comment:
Hi @micksatana -- what is this? Where is it coming from?

Reply:
Hi @JoelMarcey thanks for the review.

I added so anyone using IDE from intellij will not accidentally commit into the code